src/components/Layout/Layout.component.jsx

```diff
@@ -1,11 +1,5 @@
 import React from 'react';
-import styled from 'styled-components';
-
-const AllDiv = styled.div`
-  -webkit-box-flex: 1;
-  flex-grow: 1;
-  height: 100%;
-`;
+import AllDiv from './Layout.styles';
```

Comment:
I think you are using `*.styled` for the rest of your css files

```suggestion
import AllDiv from './Layout.styled';
```